types/next-auth/index.d.ts

```diff
@@ -28,6 +28,7 @@ interface InitOptions {
     events?: Events;
     useSecureCookies?: boolean;
     cookies?: Cookies;
+    theme?: string;
```

Comment:
From what I see  [on the documentation](https://next-auth.js.org/configuration/options#theme) this can be only three values:
```
'light' | 'dark' | 'auto' 
```
So I think we should type here more strictly 👍🏽 
```ts
theme?: 'light' | 'dark' | 'auto' 
```

Reply:
Great! Your answer is exactly what it should be.